trigger/payloads.go

```diff
@@ -432,6 +428,22 @@ func makeCommonPayload(sub *event.Subscription, selectors []event.Selector,
 	return nil, errors.Errorf("unknown type: '%s'", sub.Subscriber.Type)
 }
 
+func getFailedTestsFromTemplate(t *task.Task) ([]task.TestResult, error) {
```

Comment:
Since you're changing the object, pass t by value rather than reference so that you don't unexpectedly change it for the caller